src/components/Tooltip.jsx

```
@@ -2,9 +2,13 @@ import PropTypes from "prop-types";
 import { Tooltip as ReactTooltip } from "react-tooltip";
 import { useState, useEffect, useRef, useId, cloneElement, isValidElement } from "react";
 
-export default function Tooltip({ content, children, id, dynamicPositioning = true }) {
-  const reactId = useId();
-  const tooltipId = id || `tooltip-${reactId}`;
+export default function Tooltip({
+  children,
+  content = "Tooltip content",
+  id = `tooltip-${useId()}`,
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)